import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreService.java

```diff
@@ -182,11 +183,8 @@ protected void store(
       reportError(e);
       throw e;
     } catch (IBackupUploader.BackupExeption e) {
-      reportBackupError(e);
-      // todo: throw exception
-      DicomServiceException serviceException = new DicomServiceException(Status.ProcessingFailure, e);
-      serviceException.setErrorComment(e.getMessage());
-      throw serviceException;
+        MonitoringService.addEvent(Event.CSTORE_BACKUP_ERROR);
+        log.error("C-STORE request failed. Trying to resend...", e);
```

Comment:
Please change log message to something like this: "Backup io processing during C-STORE request is failed."